### DISCUSSION TITLE
PixArt-256's reparameterization of DiT weights

### DISCUSSION THREAD

**Discussion by ljk628** [PixArt-alpha/PixArt-alpha#84] [General]
Congrats on the great work. It seems the reparameterization of DiT-X/2's weights as initialization a critical step for making the training working for 256 resolution. Could you share the code for this part as well? This would be very helpful for getting similar improvement from other architectures. Thanks!

**Comment by kaleidoscopical** [comment]
@lawrence-cj Could you provide a sample code to do the reparameterization?

**Comment by shyshuai90** [comment]
+1
